server/src/main/java/io/spine/server/tenant/TenantIndex.java

```diff
@@ -45,6 +46,21 @@
      */
     Set<TenantId> getAll();
 
+    /**
+     * Applies the given {@code operation} to all the tenant IDs currently present in the index
+     * and those added in future.
+     *
+     * <p>The operation is applied to {@linkplain #getAll() all present tenant IDs} at once. When
+     * new IDs are {@linkplain #keep(TenantId) added} to the index, the operation is applied
+     * to them as well.
+     *
+     * <p>Note that the operation may be applied to a single {@code TenantId} more then once. In
+     * general, the caller may rely on at-least-once behavior.
+     *
+     * @param operation the operation to apply to all the tenant IDs
+     */
+    void forEachTenant(TenantIdConsumer operation);
```

Comment:
Assuming we have `getAll()` which returns a set of all tenant IDs, why do we need this method? Why isn't `for` operator enough?